src/euphorie/client/notifications/base.py

```diff
@@ -43,12 +47,24 @@ def recipient(self):
     def index(self):
         """The mail text."""
         full_name = self.webhelpers.get_user_fullname(self.account)
-        session_links = "\n".join(
-            [
-                f"* [{session.title}]({session.absolute_url()}/@@start)"
-                for session in self.sessions
-            ]
-        )
+
+        session_links = ""
+        for session in self.sessions:
+            session_url = ""
+            try:
+                session_url = session.absolute_url()
+            except ValueError:
+                logger.warn(
```

Comment:
Isn't it better to have a:
```suggestion
                logger.exception(
```
How this inconsistency can be fixed?

Reply:
done.